CONFIGURATION.md

```diff
@@ -25,7 +25,7 @@ socket.blocking.max.ms                   |  *  | 1 .. 60000      |          1000
 socket.send.buffer.bytes                 |  *  | 0 .. 100000000  |             0 | Broker socket send buffer size. System default is used if 0. <br>*Type: integer*
 socket.receive.buffer.bytes              |  *  | 0 .. 100000000  |             0 | Broker socket receive buffer size. System default is used if 0. <br>*Type: integer*
 socket.keepalive.enable                  |  *  | true, false     |         false | Enable TCP keep-alives (SO_KEEPALIVE) on broker sockets <br>*Type: boolean*
-socket.nagle.disable                     |  *  | true, false     |         false | Disable the Nagle algorithm (TCP_NODELAY). <br>*Type: boolean*
+socket.nagle.disable                     |  *  | true, false     |         false | Disable the Nagle algorithm (TCP_NODELAY) on broker sockets. <br>*Type: boolean*
```

Comment:
i know this isn't the time or place, but while I remember, a bit more detail/clarity on the `queue.buffering.backpressure.threshold` config parameter would be good. i've noticed people don't understand it.